packages/cli/src/commands/buildHandler.js

```diff
@@ -23,6 +23,15 @@ export const handler = async ({
   prisma = true,
   prerender,
 }) => {
+  recordTelemetryAttributes({
+    command: 'build',
```

Comment:
Just a note to others, @Josh-Walker-GM and I went back and forth about importing this from the corresponding file (in this case, `build.js`). I decided against it cause the last thing I want is a circular import. The tradeoff is overhead for us—if we ever rename  a command, we have to change it here too.

Reply:
and the fact that I also just hard coded the values for commands nested in directories like `setup auth dbAuth` then it did just feel a little silly to import for the simpler cases.